### ISSUE TITLE
Getting kodi working. 

### ISSUE THREAD

**Issue by stanyertom** [Gameboypi/SPW#51]
Hi I’m trying to get kodi to run on my build and wondering if anyone has had any joy with it. It just loads onto a black screen. Needs to restart to exit it. Any help would be much appreciated

**Comment by mackemint** [comment]
Did you install it like [this](https://retropie.org.uk/docs/KODI/)?

**Comment by stanyertom** [comment]
Yes installed through the menu if I remember I tried it through command line aswel and it didn’t work. And I left it in ports menu if that makes a diff. So does it work on your spw?
